src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -247,6 +247,16 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       System.err.println("Compiler runtime: " + (stop - start) + "ms.");
     }
 
+    String[] lines = {
+        "build_time=" + Long.toString(stop - start)
+    };
+    Path statsfilePath = FileSystems.getDefault().getPath(ops.statsfile);
+    try {
+        Files.write(statsfilePath, Arrays.asList(lines));
```

Comment:
Why do we need an array of we convert it to a list? Maybe just create a list two lines above?